RMSwiftUI/RMSwiftUI/ContentView.swift

```diff
@@ -7,18 +7,280 @@
+            static let yearR = "Год выпуска"
+            static let yearChar = "г."
+            static let insurance = "Страховка"
+            static let characteristics = "Характеристики"
+            static let options = "Дополнительные услуги"
+            static let credit = "Автокредит"
+            static let warning = "Предупреждение"
+            static let cancel = "Отменить"
+            static let done = "Ок"
+            static let message = "Страховка не входит в стоимость авто"
+            static let emptyString = ""
+        }
+        enum Numbers {
+            static let maxTerm = 60.0
+            static let imageSize: CGFloat = 300
+            static let monthInYear = 12
+        }
+    }
+    
+    //MARK: - Private Enum
+    private enum CarColor: CaseIterable, Equatable {
+        
+        case red
+        case green
+        case blue
+        
+        var sign: String {
+            switch self {
+            case .red:
+                return "R"
+            case .green:
+                return "G"
+            case .blue:
+                return "B"
+            }
+        }
+        
+        var name: String {
+            switch self {
+            case .red:
+                return "Красный"
+            case .green:
+                return "Зеленый"
+            case .blue:
+                return "Синий"
+            }
+        }
+    }
+    
+    
+    // MARK: - Body
+    
     var body: some View {
-        VStack {
-            Image(systemName: "globe")
-                .imageScale(.large)
-                .foregroundColor(.accentColor)
-            Text("Hello, world!")
+        
+        NavigationView {
+            Form {
+                Section{
+                    carPicker
+                }
+                Section{
+                    colorPicker
+                }
+                Section(
+                    content: {
+                        carInfo
+                    }, header: {
+                        Text(Constants.Texts.characteristics).font(.title3).bold()
+                    })
+                Section(
+                    content: {
+                        addOpions
+                    }, header: {
+                        Text(Constants.Texts.options).font(.title3).bold()
+                    })
+                Section {
+                    totalPrice
+                }
+            }
+            .navigationTitle(Constants.Texts.catalog)
+            .toolbar {
+                shareButton
+            }
+        }
+    }
+    // MARK: - Private @State
+    
+    @State private var month = 0
+    @State private var carIndex = 0
+    @State private var currentColor: CarColor = .red
+    @State private var isBuyButon = false
+    @State private var isInsurance = false
+    @State private var isInsuranceAlertShown = false
+    @State private var isSharePresented = false
+    
+    // MARK: - Private Properties
+    
+    private var insurance = 500
```

Comment:
```suggestion
    private var insurance =
```
начальное значение берем из констант